t/23-baseclass.t

```diff
@@ -308,6 +309,57 @@ subtest 'SSH utilities' => sub {
     };
 };
 
+sub _prepare_video_encoder ($baseclass) {
+    my @pipes;
+    for (1 .. 3) {
+        my $pipe = IO::Pipe->new;
+        my $pid = fork;
+        if ($pid) { $pipe->writer }
+        elsif (defined $pid) {
+            $pipe->reader;
+            my @lines = <$pipe>;
+            exit;
```

Comment:
These lines will likely not be tracked so they should likely be marked as uncoverable.

Reply:
ok. we're not collecting coverage for `t/` yet, but I marked the lines and hope I got it right